concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -156,19 +156,19 @@ Gauchat-1925-480-154	154	The circle	Le cercle	cĭrculu	52	1467	CIRCLE
 Gauchat-1925-480-155	155	is round	est rond	rotŭndu	52	1395	ROUND
 Gauchat-1925-480-156	156	The wheel	La roue	rŏta	52	710	WHEEL
 Gauchat-1925-480-157	157	(is) round (feminine)	(est) ronde	rotŭnda	54		
-Gauchat-1925-480-158	158	One must	Il faudra	fallere + habet	54		
+Gauchat-1925-480-158	158	One will have to	Il faudra	fallere + habet	54		
 Gauchat-1925-480-159	159	soon	bientôt	bĕne tŏstu	54	703	SOON
-Gauchat-1925-480-160	160	mow	tondre	tŏndere	54	190	MOW
-Gauchat-1925-480-161	161	the ewes	les brebis	fētas	54	1345	EWE
-Gauchat-1925-480-162	162	sharpen (?)	racérer	reaciariare	54		
+Gauchat-1925-480-160	160	shear	tondre	tŏndere	54	1532	SHEAR
+Gauchat-1925-480-161	161	the sheep	les brebis	fētas	54	1331	SHEEP
+Gauchat-1925-480-162	162	scrape	racérer	reaciariare	54	1767	SCRAPE
 Gauchat-1925-480-163	163	the axe	la hache	happia (+ itta)	56	677	AXE
-Gauchat-1925-480-164	164	beat in the barn	battre en grange	excŭtere	56		
+Gauchat-1925-480-164	164	thresh in the barn	battre en grange	excŭtere	56		
```

Comment:
Maybe 285 THRESH, depending of course how closely the context should be tied to the glosses.

Reply:
it is a phrase, so difficult to elicit here.